pages/services/edge-lb/1.1/pool-configuration/index.md

```diff
@@ -12,18 +12,20 @@ This is a reference for all possible Edge-LB pool configuration values, as well
 
 # Configuration Reference
 
-The configuration reference containing all possible options and descriptions of each can be found by running the following command:
+The API reference, in swagger format, containing all possible options and short descriptions of each endpoint can be found by running the following command:
 
 ```
 dcos edgelb show --reference
 ```
 
 For more information, see the [CLI Reference Guide entry for `dcos edgelb show`](/1.1/cli-reference/dcos-edgelb-show/).
 
+Choose an API version at the left to view the appropriate configuration reference or examples.
+
 # API Versions
 
-A new top level configuration field named `apiVersion` was introduced in Edge-LB v1.1.0. The two models are almost identical with one important difference: `pool.haproxy.backends.servers` (apiVersion `V1`) has been replaced with `pool.haproxy.backends.services` with a more intuitive way to select services/backends for HAProxy.
+A new top level pool configuration field named `apiVersion` was introduced in Edge-LB v1.0.0. The two models are almost identical, with one important difference: `pool.haproxy.backends.servers` (in apiVersion `V1`) has been replaced with `pool.haproxy.backends.services`, with a more intuitive way to select services/backends for HAProxy.
 
-**Note:** For backwards compatibility, the `apiVersion` field defaults to `V1`. Hence, in order to use the `V2` config, you must explicitly set the `pool.apiVersion` to `"V2"`.
+**Note:** Edge-LB 1.0 and later supports both the `V1` and `V2` API for backwards compatibility. Therefore clients that were written against Edge-LB versions prior to Edge-LB 1.0 should work without any modifications with Edge-LB 1.0 and later. New setups should use api `V2` as at some point `V1` is going to be deprecated and then removed.
 
-Choose an API version at the left to view the appropriate configuration reference or examples.
+**Note:** The `apiVersion` field in th pool definition defaults to `V2` if it was not provided. Hence, in order to use the `V1` config, you must explicitly set the `pool.apiVersion` to `"V1"`.
```

Comment:
This would be a breaking change as the earlier default was `V1`. We might wrongly assume a `V1` app definition to `V2`. We need to follow up with the appropriate error message

Reply:
This change only makes the documentation reflect the current state. The V1 seems to have never been a default:

https://github.com/mesosphere/dcos-edge-lb/blob/master/framework/edgelb/cli/dcos-edgelb/commands/pool.go#L692-L700
https://github.com/mesosphere/dcos-edge-lb/blob/master/framework/edgelb/cli/dcos-edgelb/commands/pool.go#L710-L721

The API server itself has a very clear distinction wrt. API versions - the version is determined by the URL path (`/V1/blah...` vs `/V2/blah...`). So if you were using CLI and did not specify api version, you were using V2.